lib/dotcom/system_status/groups.ex

```diff
@@ -0,0 +1,465 @@
+defmodule Dotcom.SystemStatus.Groups do
+  @moduledoc """
+
+  A module that groups alerts into statuses for the system status
+  widget. See `Dotcom.SystemStatus` for more information.
+
+  """
+
+  alias Alerts.Alert
+
+  # @lines is sorted in the order in which `groups/2` should sort its results
+  @lines ["Blue", "Orange", "Red", "Green"]
+  @green_line_branches ["Green-B", "Green-C", "Green-D", "Green-E"]
+
+  @doc """
+
+  Returns a data structure that can be used in the system status
```

Comment:
We don't use this empty line in docs anywhere else. We shouldn't describe this module as being for use in front-end component. It should be useful on its own. Something like `takes alerts groups them in order to communicate system status of each route`.

Reply:
Bye bye empty lines!

I could have sworn I saw it used somewhere else, but now I can't find it, and my editor plays more nicely when there is that empty line, but I also agree that it looks weird (and is inconsistent with the rest of the dotcom world)